go/felix/proto/felixbackend.proto

```diff
@@ -145,8 +145,9 @@ message PolicyID {
 }
 
 message Policy {
-  repeated Rule inbound_rules = 1;
-  repeated Rule outbound_rules = 2;
+  bool untracked = 1;
```

Comment:
There's also another place you'll need to update: we convert Policy objects into ParsedRule objects (which contain rules modified to use ipsets instead of selectors/tags) so you'll need to pass the flag via those objects.